runafluid.cpp

```diff
@@ -196,8 +196,13 @@ void fire(ItmNs::Itm::coreprof &coreprof, ItmNs::Itm::coreimpur &coreimpur,
 	distribution_out.time = distribution_in.time+timestep;
 
 	// HDF5 export
-	if (!modules.output_path.empty()){
-			H5std_string hdf5_file_name(modules.output_path);
+	if (modules.hdf5_output == true){
+
+			std::string str_shot_number = std::to_string(shot_number);
+			std::string str_run_number = std::to_string(run_number);

```

Comment:
Should be formatted as for the MDS+ data. Can use 
http://www.cplusplus.com/reference/cstdio/printf/
http://www.cplusplus.com/reference/cstdio/sprintf/

Reply:
Done.